profile.md

```diff
@@ -37,7 +37,7 @@ _Authored by the WLCG AuthZ Working Group_
 
 # Introduction
 
-This document describes how WLCG users may use the available geographically distributed resources without X.509 credentials.  In this model, clients are issued with bearer tokens; these tokens are subsequently used to interact with resources.  The tokens may contain authorization groups and/or capabilities, according to the preference of the VO, applications and relying parties. 
+This document describes how WLCG users may use the available geographically distributed resources without X.509 credentials.  In this model, clients are issued with bearer tokens; these tokens are subsequently used to interact with resources.  The tokens may contain authorization groups and/or capabilities, according to the preference of the virtual organization (VO), applications and relying parties.  Each VO acts as an _attribute authority_ with its own policies on the issuance and contents of tokens.
```

Comment:
I believe that, currently, WLCG JWT Profile assumes/requires each issuer supports precisely one VO.

Reply:
The description of the `iss` claim in the Common Claims subsection says:

`For WLCG this would be the VO.`

While I think we will need to say more about this topic, it would seem best to wait for outcomes from the Grand Unified Token Profile WG and then adjust our profile where desirable, to allow it to be used as a reference also in contexts that are incompatible with the current wordings in this document.